### ISSUE TITLE
ApplicationInsights not working

### ISSUE THREAD

**Issue by martincostello** [martincostello/alexa-london-travel#45]
### Expected behaviour

Skill usage is reported to ApplicationInsights.

### Actual behaviour

Requests to the skill's Lambda function fail with this error:

```
{
  "errorMessage": "Cannot find module 'diagnostic-channel'",
  "errorType": "Error",
  "stackTrace": [
    "Function.Module._load (module.js:276:25)",
    "Module.require (module.js:353:17)",
    "require (internal/module.js:12:17)",
    "Object.<anonymous> (/var/task/node_modules/applicationinsights/out/AutoCollection/CorrelationContextManager.js:4:28)",
    "Module._compile (module.js:409:26)",
    "Object.Module._extensions..js (module.js:416:10)",
    "Module.load (module.js:343:32)",
    "Function.Module._load (module.js:300:12)",
    "Module.require (module.js:353:17)",
    "require (internal/module.js:12:17)",
    "Object.<anonymous> (/var/task/node_modules/applicationinsights/out/applicationinsights.js:2:33)",
    "Module._compile (module.js:409:26)",
    "Object.Module._extensions..js (module.js:416:10)",
    "Module.load (module.js:343:32)",
    "Function.Module._load (module.js:300:12)"
  ]
}
```

### Steps to reproduce

Invoke the skill.

See #41, #43 and #44.

**Comment by martincostello** [comment]
Manually uploading the skill code as a ZIP fixes the error above, so seems to be an issue with dependencies either not being resolved in Travis, or Travis not uploading the dependencies in `npm_modules` correctly.

However, if ApplicationInsights is enabled with the manual fix, it still hangs.

**Comment by martincostello** [comment]
#46 didn't fix it either :(

**Comment by martincostello** [comment]
Dependency error fixed by #47, but hang still not resolved.

**Comment by martincostello** [comment]
This has been _partially_resolved by #59. Now it doesn't hang if used frequently, but the first request after a period of inactivity seems to cause the problem to come back.

**Comment by martincostello** [comment]
#62 didn't fix the flakiness either.

**Comment by martincostello** [comment]
It looks like _something_ in #64 fixed this.

**Comment by martincostello** [comment]
Fixed in [1.0.9](https://github.com/martincostello/alexa-london-travel/releases/tag/v1.0.9). Exact root cause not understood.
